### ISSUE TITLE
Rails 5.2.2.1 not yet supported

### ISSUE THREAD

**Issue by nick-adams** [rootstrap/active-storage-base64#26]
I've been using this gem and it's working great. Thanks for all your work on it! 

However, I did recently try to upgrade Rails (there was a security release 3/13: https://weblog.rubyonrails.org/2019/3/13/Rails-4-2-5-1-5-1-6-2-have-been-released/) but Bundler threw an error because this gem has a hard dependency on `<= 5.2.2`

```Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 5.2.2.1)

    active_storage_base64 was resolved to 0.1.3, which depends on
      rails (~> 5.2.0, <= 5.2.2)
```  

I just wanted to bring this to your attention and see if there was a plan or preference on how to update things to allow for this upgrade. 

Disclaimer: I am not a seasoned Rails developer, so I don't know the exact protocol on things like this. I wanted to reach out before submitting a PR that just bumped the Rails dependency in the gemspec in fear of other consequences. I could certainly do that though if that's the recommendation.

**Comment by archonic** [comment]
Seems like this should just be `rails (~> 5.2)`.

**Comment by archonic** [comment]
Fixed this in #27

**Comment by skcc321** [comment]
Thank for the fix. When do you expect to release the change?

**Comment by archonic** [comment]
That's up to @nick-adams, I don't own the original repo.

**Comment by Ricoch** [comment]
Hello there, thanks for posting the issue.
Just some insight into why we limited to these versions of Rails. At the moment of the creation of this gem, the current maximum rails version was 5.2.2, and we knew that in their master branch they were working on some changes that were not compatible at all with the gem in it's current version. We didn't know at that point wether those changes were going to be for the next version (which was 5.2.2.1) or if they were going to come on Rails 6 (which seems to be the case), therefore we decided to lock the versions up to 5.2.2. Thanks for noticing!
